wpengine-geoip.php

```diff
@@ -633,7 +633,13 @@ public function action_admin_init_check_plugin_dependencies() {
 	public function action_admin_notices() {
 		foreach ( $this->admin_notices as $type => $notices ) {
 			foreach ( $notices as $key => $notice ) {
-				echo wp_kses( "<div class=\"notice notice-{$type} wpengine-geoip is-dismissible\" data-key=\"{$key}\"><p>$notice</p></div>" );
+				?>
+				<div class="notice notice-<?php echo sanitize_text_field( $type ); ?> wpengine-geoip is-dismissible" data-key="<?php echo sanitize_text_field( $key ); ?>">
```

Comment:
meh. sanitize_text_string did basically what I needed. You're right. Switching them to esc_attr and esc_html.